Description: Revert module version zeroing
 Trying to revert the zeroing since the crash seems to be related to 
 the version number. 

Index: qtpim/.qmake.conf
===================================================================
--- qtpim.orig/.qmake.conf
+++ qtpim/.qmake.conf
@@ -2,4 +2,4 @@ load(qt_build_config)
 
 DEFINES += QT_NO_JAVA_STYLE_ITERATORS
 
-MODULE_VERSION = 0.0.0
+MODULE_VERSION = 5.0.0
